### ISSUE TITLE
Lazyload wont work on aws s3 hosted images

### ISSUE THREAD

**Issue by GabrielTheCoder** [jassa/lazyload-rails#14]
title pretty much says it all. the grey square will stay if you try to apply the lazyload helper on dynamic images like

`<%= image_tag(@post.image1, lazy: true) %>`

images are stored via paperclip on aws s3.

**Comment by GabrielTheCoder** [comment]
no, unfortunately its not possible to lazyload external hosted images

**Comment by jamesachilles** [comment]
Has anyone had any luck with S3 hosted images? or any update here? Thanks

**Comment by petitkriket** [comment]
no luck either, has anyone know of an alternative or work around ?

**Comment by gabrielboeker** [comment]
use a different library. the gem is pretty neat except of that.

**Comment by roaldjap** [comment]
have you initialized your lazyload(); on this? 

I believe you need this to trigger it so it would "lazyload" your images.

just wondering what would be the compiled version of `<%= image_tag(@post.image1, lazy: true) %>`

I was just thinking `@post.image1` would return some sort of url right?
